pwiz_tools/Skyline/Model/Lib/BiblioSpecLite.cs

```diff
@@ -1168,13 +1156,19 @@ private bool Load(ILoadMonitor loader, IProgressStatus status, bool cached)
                 // should be suppressed because we know Skyline is going to try again.
                 if (!cached)
                 {
-                    var failureException = new Exception(FormatErrorMessage(x), x);
-                    if (ExceptionUtil.IsProgrammingDefect(x))
+                    // SQLiteExceptions are not considered programming defects and should be shown to the user
+                    // as an ordinary error message
+                    if (x is SQLiteException || !ExceptionUtil.IsProgrammingDefect(x))
+                    {
+                        var message = string.Format(Resources.BiblioSpecLiteLibrary_Load_Failed_loading_library__0__, FilePath);

```

Comment:
I keep asking for a separate message for when the FilePath does not exist. Then it would be helpful, I think, to report that it is missing instead of just the we "Failed loading library". It would be really easy to do that test and have 2 messages.